src/main/kotlin/com/github/patou/gitmoji/GitMojiConfig.kt

```diff
@@ -135,6 +220,10 @@ class GitMojiConfig(private val project: Project) : SearchableConfigurable {
         includeGitMojiDescriptionConfig = props.getBoolean(CONFIG_INCLUDE_GITMOJI_DESCRIPTION, false)
         textAfterUnicodeConfig = props.getValue(CONFIG_AFTER_UNICODE, " ")
         languagesConfig = props.getValue(CONFIG_LANGUAGE, "auto")
+        gitmojiJsonUrlConfig = props.getValue(CONFIG_GITMOJI_JSON_URL, CONFIG_GITMOJI_JSON_URL_DEFAULT)
+        localizationUrlConfig = props.getValue(CONFIG_LOCALIZATION_URL, CONFIG_LOCALIZATION_URL_DEFAULT)
```

Comment:
Maybe the default value must not be filled ?
in custom mode I thinks it's better to not display a default value.

What to you thinks about it ?